### PR TITLE
added note re: sticky sessions [issue 568]

### PR DESCRIPTION
**Pull request by tpoitras**
added note as per github issue 568

https://github.com/openshift/openshift-docs/issues/568

### CODE REVIEW

File: architecture/core_concepts/routes.adoc [hunk 1]
```diff
@@ -125,6 +125,13 @@ Specific configuration for this router implementation is stored in the
 *_haproxy-config.template_* file located in the *_/var/lib/haproxy/conf_*
 directory of the router container.
 
+[NOTE]
+====
+The  `balance source` directive does not distinguish between external client IP
+addresses. Unless the router is running with hostNetwork: true, all external
```
Comment (adellape) [inline architecture/core_concepts/routes.adoc]:
```
`*hostNetwork: true*`
```

**Comment by adellape** [comment]
@pweil- Can you review? (Looks like your doppelganger was tagged in the original Issue)

**Comment by pweil-** [comment]
@adellape the wording is a little misleading.  It's not that `balance source` is failing to distinguish the IPs, it is given the same IP due to the NATing if hostNetwork is not use (@ramr can confirm that).  :+1: to having this called out though.


> Looks like your doppelganger was tagged

I should've been more creative with my github name...created before I had a job that actually required me to use github :smile:  I wonder how much mail that user is deleting.  Sorry doppelganger!

**Comment by ramr** [comment]
@adellape @pweil-  yeah agree, the wording could make it a little misleading. The behavior of balance source in the ```--host-network=false``` haproxy router case is because of the originating IP (haproxy remote) being the same. And just fyi, there is some work in k8s upstream to use iptables in lieu of proxying, which would make ```balance source```  behave as intended (when using the container network for the haproxy router).
So yeah, calling it out is good :+1 but mentioning why would be better.

**Comment by tpoitras** [comment]
Thank you, @ramr & @pweil- & @adellape :rocket: 

Updated based on your comments. Good to merge now, @adellape ?

**Comment by adellape** [comment]
LGTM

**Pull request merged**